ddtrace/ext/ci.py

```diff
@@ -203,8 +197,9 @@ def extract_azure_pipelines(env):
         JOB_URL: job_url,
         git.REPOSITORY_URL: env.get("SYSTEM_PULLREQUEST_SOURCEREPOSITORYURI") or env.get("BUILD_REPOSITORY_URI"),
         git.COMMIT_SHA: env.get("SYSTEM_PULLREQUEST_SOURCECOMMITID") or env.get("BUILD_SOURCEVERSION"),
-        git.BRANCH: branch,
-        git.TAG: tag,
```

Comment:
we no longer set tag at all now? Is that expected when the job runs for a tag?

Reply:
(The PR description and title says that we keep both)

Reply:
Some providers don't have separate tag and branch variables and instead set branch to be `refs/heads/tags/v1.0` for tags. The logic to detect that was previously inside the code specific to each provider, but now we just set it as `git.BRANCH` here and we have centralized logic to use the branch as tag when needed.

Reply:
👍 thanks for the clarification!